roles/kubernetes/control-plane/tasks/kubelet-fix-client-cert-rotation.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Fixup kubelet client cert rotation 1/2
+  lineinfile:
+    dest: "{{ kube_config_dir }}/kubelet.conf"
```

Comment:
Why don't we use `path:` instead?
According to the following note of https://docs.ansible.com/ansible/latest/collections/ansible/builtin/lineinfile_module.html
```
As of Ansible 2.3, the dest option has been changed to path as default, but dest still works as well.
```
it seems better to use path and Kubespray requires ansible v2.9.17 at this time.

Reply:
force of habit / copy paste, will update, thanks for the review